examples/tools/scene_viewer.rs

```diff
@@ -0,0 +1,407 @@
+use bevy::{
+    asset::{AssetServerSettings, LoadState},
+    input::mouse::MouseMotion,
+    prelude::*,
+    render::{
+        camera::{Camera2d, Camera3d, CameraProjection},
+        primitives::{Aabb, Frustum, Sphere},
+    },
+    scene::InstanceId,
+};
+
+fn main() {
+    println!(
+        "
+Controls:
+    WSAD   - forward/back/strafe left/right
+    LShift - 'run'
+    E      - up
```

Comment:
This is a strange choice of control for this. Why not Space / LCtrl for up /down?

Reply:
This isn't strange if one considers that both Unity and Unreal Engine both use E/Q for up/down.

Reply:
SGTM.